### ISSUE TITLE
[BUG]: Matching issues again - Waybig

### ISSUE THREAD

**Issue by ScoBoXxX** [CodyBerenson/PGMA-Modernized#242]
# Mandatory #

## Describe the bug: ## 
Looks like Waybig is having some issues again on certain files.  Mine seem to be Men.com titles at least that is all I am noticing.  Here are 2 titles that are failing:
(Men.com) - Malik Delgaty Creampies Fellow Bodybuilder Clark Reid In ‘FUCKING JACKED’ (2023).mp4
(Men.com) - LEO LOUIS And ENZO MÜLLER Fuck LEO BACCHUS In ‘Beaux Are Hoes, Part 2’ (2023).mp4

## Specific Agent(s) Causing the Issue: ##
Waybig

## Index Site URL(s) Attempting to Match: ##
https://www.waybig.com/blog/2023/05/15/men-malik-delgaty-creampies-fellow-bodybuilder-clark-reid-fucking-jacked/
https://www.waybig.com/blog/2023/05/09/men-leo-louis-enzo-muller-fuck-leo-bacchus-beaux-are-hoes-part-2/

## Log Attached: ##
YES

# Optional #

## Desktop (please complete the following information): ##
 - OS: Windows 10
 - Browser Edge (latest)
 - Version 113.0.1774.42 (Official build) (64-bit)

[com.plexapp.agents.WayBig.log](https://github.com/CodyBerenson/PGMA-Modernized/files/11479646/com.plexapp.agents.WayBig.log)

**Comment by JPH71** [comment]
@ScoBoXxX

Please update your PGMA bundles... Waybig is currently on v.2019.12.22.41 -
you are on v.2019.12.22.39
The issue you describe was sorted by the latest updates.

Jason xxx

**Comment by ScoBoXxX** [comment]
Thanks Jason - I updated but am still having the same issue.  It seems to fail rather quickly as I can see the spinner start and stop real quick.  I attached a new log.


[com.plexapp.agents.WayBig.log](https://github.com/CodyBerenson/PGMA-Modernized/files/11490167/com.plexapp.agents.WayBig.log)

**Comment by CodyBerenson** [comment]
Hi @ScoBoXxX 

You're receiving an error message:  Error: <QUIT: Scraping Parameters Not Set Up!>

Please check and update each agent's settings parameters.  Depending on your system, you may need to then stop and restart your PMS.

**Comment by ScoBoXxX** [comment]
I re-downloaded the newest code again and I'm still getting this scraping error.  I'm not sure what all would have changed in the parameters.  I never change them.  I uploaded a new log.  If you could give me an idea of what to look at in settings/parameters I am happy to.  But again - all I did was copy the new PGMA files over to Plug-ins folder.
[com.plexapp.agents.WayBig.log](https://github.com/CodyBerenson/PGMA-Modernized/files/11492498/com.plexapp.agents.WayBig.log)

**Comment by CodyBerenson** [comment]
I matched relentlessly last night on Waybig, no issues.  So, after you reposted your log I tried matching a new couple of titles.  I got a similar (but different) error to yours, but similar in that it didn't even start the scraping process...just like yours.  I noticed something odd with the token though:  On the agents where I *don't* have the token as part of the parameters, they not only work fine, but they are using obtaining a token *different* than my usual plex token....  

So, I wiped out the token from my WayBig Agent settings, shut down PMS, restarted, and Agent fires up as expected, starts scraping and the new titles match fine.  Maybe you want to give that a try?

**Comment by ScoBoXxX** [comment]
Hey Cody.  I don't have any tokens in my parameter - except for GEVI.  I don't really recall ever having any tokens in the settings for Waybig or others but maybe?  I don't really change anything in those settings so am not really sure why they'd have changed.

But mine are all blank.  Maybe I should try adding one - where would I get that token from?  The one in GEVI is mDGjkY3i3DbfhD_2k8ay but again I have no idea where that came from.

**Comment by JPH71** [comment]
You need to update the preferences for each of the scrapers then re-start
plex.
This is why you get You're receiving an error message: Error: <QUIT:
Scraping Parameters Not Set Up!>
You need to go into your preferences...... as Cody mentioned to you....

Just like previously when a Thumbor Address changed - one had to go into
the preferences and change then save the preferences...

**Comment by CodyBerenson** [comment]
@ScoBoXxX 

Please confirm that you copied ALL of the latest release, and not just the WayBig.bundle.  

I was able to recreate your issue and raise the same error by only updating the WayBig.bundle from the last release and not the _PGMA folder as well.

**Comment by ScoBoXxX** [comment]
I made that mistake once a while back - never again.  I copied it all over.

**Comment by JPH71** [comment]
Here is the problem as shown by your log file

2023-05-16 15:07:42,025 (2910) :  INFO (logkit:16) - WayBig  - START :: 1.
matchiafdduration        Default = true       Set As = True
2023-05-16 15:07:42,025 (2910) :  INFO (logkit:16) - WayBig  - START :: 2.
matchsiteduration        Default = false      Set As = False
2023-05-16 15:07:42,025 (2910) :  INFO (logkit:16) - WayBig  - START :: 3.
durationdx               Default = 0          Set As = 0
2023-05-16 15:07:42,025 (2910) :  INFO (logkit:16) - WayBig  - START :: 4.
detect                   Default = false      Set As = False
2023-05-16 15:07:42,025 (2910) :  INFO (logkit:16) - WayBig  - START :: 5.
usebackgroundart         Default = true       Set As = True
2023-05-16 15:07:42,040 (2910) :  INFO (logkit:16) - WayBig  - START :: 6.
prefixlegend             Default = true       Set As = True
2023-05-16 15:07:42,040 (2910) :  INFO (logkit:16) - WayBig  - START :: 7.
plextoken                Default =            Set As = ********
2023-05-16 15:07:42,040 (2910) :  INFO (logkit:16) - WayBig  - START :: 8.
resetmeta                Default = false      Set As = False
2023-05-16 15:07:42,040 (2910) :  INFO (logkit:16) - WayBig  - START :: 9.
prefixgenre              Default = false      Set As = False
2023-05-16 15:07:42,040 (2910) :  INFO (logkit:16) - WayBig  - START :: 10.
postersourcedownload     Default = Local - No Set As = Local - No
2023-05-16 15:07:42,040 (2910) :  INFO (logkit:16) - WayBig  - START :: 11.
systemcollection         Default = true       Set As = False
2023-05-16 15:07:42,040 (2910) :  INFO (logkit:16) - WayBig  - START :: 12.
genrecollection          Default = Purple     *Set As = None*
2023-05-16 15:07:42,040 (2910) :  INFO (logkit:16) - WayBig  - START :: 13.
countrycollection        Default = true       Set As = False
2023-05-16 15:07:42,040 (2910) :  INFO (logkit:16) - WayBig  - START :: 14.
countrypostertype        Default = Map        Set As = Map
2023-05-16 15:07:42,040 (2910) :  INFO (logkit:16) - WayBig  - START :: 15.
studiocollection         Default = true       Set As = False
2023-05-16 15:07:42,040 (2910) :  INFO (logkit:16) - WayBig  - START :: 16.
seriescollection         Default = true       Set As = False
2023-05-16 15:07:42,040 (2910) :  INFO (logkit:16) - WayBig  - START :: 17.
directorcollection       Default = true       Set As = False
2023-05-16 15:07:42,040 (2910) :  INFO (logkit:16) - WayBig  - START :: 18.
castcollection           Default = true       Set As = False

In each agent - GEVI, AEBN through to Wolfe - if you are using Genre
collections select a different colour click save then click the colour you
want and click save.
If not select *No*

Restart your PMS
Scrape first title....
look at this part of the log....

**Comment by CodyBerenson** [comment]
@ScoBoXxX  I'm not sure what else to tell you.  You and I both have Windows systems.  

1. I was able to rrecreate your issue by having the incorrect _PGMA folder.  Jason just confimed offline to me that he too believes this to be the culprit.  
2. I then deleted everything in the plugins folder, reapplied the latest release in its entiretly, stopped and restarted PMS
3. I rescanned your two titles and the results are as expected

![image](https://github.com/CodyBerenson/PGMA-Modernized/assets/40373801/13c1558a-7d72-405e-9fd2-be6e36f7bc32)

Why don't you try deleting (not copying and replacing) the contents of your plugin folder.  Stop and restart PMS (or reboot your desktop).  Rescan.

**Comment by ScoBoXxX** [comment]
I'll try it again tomorrow.  I do delete the contents every time i update and then copy the unzipped updated files into the Plug-ins folder.  I'll re-download and try again tomorrow. thanks.

**Comment by ScoBoXxX** [comment]
Update: i went through the settings again and saw that Genre collection was set to Purple.  I changed it to No (I don't use that) and now the 2 files are matching fine.
I'm not sure why it was set to Purple or why it being set to Purple caused failures - but that seems to have been it.  I'll add it to the things to watch going forward.

thanks for the help again and sorry for being a pain.

sb

**Comment by CodyBerenson** [comment]
You aren't a pain, and I"m glad it is sorted out.   And, I apologize for obliviously looking right past it and not seeing it in your log earlier as Jason did when he took a look:  "None" means the new parameter had not yet been set (it wasn't actually set to Purple as you state above....that's one of the values in the dropdown but you still actually have to select a value).   

FYI, this parameter was added in the last release...here's the explanation from the release notes:

*have made some improvements to the collections section:

On my samsung tv, I could not see the collection images when they were in black. So I have now given users a set of colours - Black, Purple, Blue, Green, Red and Brown

I have set collections on my system to blue for the scenes  scrapers and films to Purple...

Selecting no colour in the preferences turns off Genre Collections

Along with these - I have downloaded some new icons to better represent the genres*
